plutus-benchmark/flat/Codec.hs

```diff
@@ -0,0 +1,71 @@
+{-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE PackageImports    #-}
+
+module Codec ( Codec(..)
+             , flatCodec
+             , cborCodec
+             , withZlib
+             , withPureZlib
+             , codecs
+             ) where
+
+import           Control.DeepSeq                     (NFData, force)
+import qualified Data.ByteString                     as BS
+import qualified Data.ByteString.Lazy                as LBS
+import           Data.Either                         (fromRight)
+import           Data.Text                           (Text)
+
+import           Language.PlutusCore                 (Name (..), Unique (..))
+import           Language.PlutusCore.Universe
+import           Language.UntypedPlutusCore
+import           Language.UntypedPlutusCore.DeBruijn
```

Comment:
The import of ‘Language.UntypedPlutusCore.DeBruijn’ is redundant
      except perhaps to import instances from ‘Language.UntypedPlutusCore.DeBruijn’